manifests/r/Ryochan7/DS4Windows/3.3.0/Ryochan7.DS4Windows.installer.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Ryochan7.DS4Windows
+PackageVersion: 3.3.0
+InstallerType: nullsoft
+Scope: machine
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
+  - PackageIdentifier: Microsoft.DotNet.DesktopRuntime.8
+  - PackageIdentifier: ViGEm.ViGEmBus
+  - PackageIdentifier: ViGEm.HidHide
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Ryochan7/DS4Windows/releases/download/v3.3.0/DS4Windows_installer_x64_v3.3.0.exe
+  InstallerSha256: 3D58356CF941FB739F24AF900A5AE9E9BFB3A18EC1986008D7D447F9DCB77BD2
```

```suggestion
  InstallerSha256: 3D58356CF941FB739F24AF900A5AE9E9BFB3A18EC1986008D7D447F9DCB77BD2
  ProductCode: DS4Windows
```

Additional metadata.